Procfile

```diff
@@ -1 +1 @@
-web: node server.js --verbose
+web: RESOURCE_INDEX=$(RESOURCE_INDEX_PREFIX="$HEROKU_BRANCH" ./get-resource-index) node server.js --verbose
```

Comment:
I find this sort of weird. Especially since for production apps (e.g. nextstrain.org, next.nextstrain.org) we need `RESOURCE_INDEX` to be calculated here to match the `config.json`. And the `config.json` in use is hardcoded everywhere, instead of respecting `CONFIG_FILE`.

What if instead of doing this just-in-time overriding of `RESOURCE_INDEX` we set it as a config var on the review app at creation (or very shortly afterwards)? This could be by moving to creating review apps ourselves as part of CI (using the [API endpoint](https://devcenter.heroku.com/articles/platform-api-reference#review-app-create)) or using the [post-deploy script](https://devcenter.heroku.com/articles/github-integration-review-apps#the-postdeploy-script) to update the app with the new config var.

Reply:
Tracking creating Heroku apps through CI in https://github.com/nextstrain/nextstrain.org/issues/854